apps/glusterfs/app.go

```diff
@@ -289,6 +300,16 @@ func (a *App) setFromEnvironmentalVariable() {
 			a.conf.KubeConfig.RebalanceOnExpansion = value
 		}
 	}
+
+	env = os.Getenv("HEKETI_MAX_LIVE_OPERATIONS")
```

Comment:
Not sure whether `LIVE` or `CONCURRENT` sound better here?

Reply:
How about `OUTSTANDING` or `INFLIGHT`?

Reply:
I like INFLIGHT because it matches how I speak about it. I don't want to use the word concurrent because we don't promise anything about concurrency in regards to this behavior.

Reply:
`INFLIGHT` looks more close to this.